src/Features/Lsif/GeneratorTest/RangeResultSetTests.vb

```diff
@@ -0,0 +1,81 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Microsoft.VisualStudio.LanguageServer.Protocol
+
+Namespace Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.UnitTests
+    <UseExportProvider>
+    Public NotInheritable Class RangeResultSetTests
```

Comment:
not sure if it would be worth having tests for more weird results, e.g. aliases, extern alias, etc

Reply:
Aliases are covered in line 20, but good call on extern aliases.